scripts/shuffle/vis.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
Can you include the visual comparison in the pr description?

Reply:
done